test/spec/modules/outbrainBidAdapter_spec.js

```diff
@@ -390,7 +390,8 @@ describe('Outbrain Adapter', function () {
                 minduration: 3,
                 maxduration: 10,
                 startdelay: 2,
-                placement: 4,
+                placement: 5,
+                plcmt: 4,
                 linearity: 1
```

Comment:
I think you simply ordered these two lines differently than your tests expected

Reply:
It passed, if you print it to console somehow you can, but I'm sure others have a better way